rules/tines_rules/tines_global_resource_destruction.py

```diff
@@ -0,0 +1,25 @@
+from global_filter_tines import filter_include_event
+from panther_base_helpers import deep_get
+from panther_tines_helpers import tines_alert_context
+
+
+def rule(event):
+    if not filter_include_event(event):
+        return False
+
+    return (
+        deep_get(event, "operation_name", default="NO_OPERATION_NAME")
```

Comment:
all defaults need to be surrounded by <> ex. default="<NO_OPERATION_NAME>") please fix this throughout the file